File: 19 poklon_za_rodjendan/poklon_za_rodjendan.cpp
#include <iostream>

using namespace std;

int main(int argc, char const *argv[]) {
  double n;
  cin >> n;
  int m;
  cin >> m;
  int brojac = 0;
  for (int i = 0; i < m; i++) {
    double cenaProizvoda;
    cin >> cenaProizvoda;
    if (cenaProizvoda <= n)
      brojac++;
  }
  cout << brojac << endl;

  return 0;
}

File: 19 poklon_za_rodjendan/poklon_za_rodjendan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace poklon_za_rodjendan
{
    class Program
    {
        static void Main(string[] args)
        {
            double n = double.Parse(Console.ReadLine());
            int m = int.Parse(Console.ReadLine());
            int brojac = 0;
            for (int i = 0; i < m; i++) {
                double cenaProizvoda = double.Parse(Console.ReadLine());
                if (cenaProizvoda <= n)
                    brojac++;
            }
            Console.WriteLine(brojac);
        }
    }
}
